playwright.config.ts

```diff
@@ -48,13 +48,18 @@ export default defineConfig<PluginOptions>({
     // 2. Run tests in Google Chrome. Every test will start authenticated as admin user.
     {
       name: 'chromium',
-      use: { ...devices['Desktop Chrome'], storageState: 'playwright/.auth/admin.json' },
+      use: {
+        ...devices['Desktop Chrome'],
+        storageState: 'playwright/.auth/admin.json',
+        // Used by the Copy URL test
+        permissions: ['clipboard-read', 'clipboard-write'],
+      },
       dependencies: ['auth'],
     },
   ],
 });
 
-function getGrafanaUrl() {
+export function getGrafanaUrl() {
```

Comment:
```suggestion
function getGrafanaUrl() {
```

I think it's only used in this file, right?

Reply:
no, its used for the `Copy url` e2e test as well that way we have a consistent grafana url also for the tests

Reply:
You're right, sorry I missed it!